backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Services/ConsoleAppHub/ProcessReceivedDataService.cs

```diff
@@ -5,93 +5,109 @@ namespace Squirrel.SqlService.BLL.Services.ConsoleAppHub;
 
 public class ProcessReceivedDataService : IProcessReceivedDataService
 {
-    // TODO: Implement all functions to process data received from ConsoleApp 
+    private readonly ResultObserver _resultObserver;
 
-    /// <summary>
-    /// Just for debugging and demo
-    /// </summary>
-    private Task ShowResult(string clientId, QueryResultTableDTO queryResultTableDTO)
+    public ProcessReceivedDataService(ResultObserver resultObserver)
+    {
+        _resultObserver = resultObserver;
+    }
+    private Task ShowResult(Guid httpId, QueryResultTableDTO queryResultTableDTO)
```

Comment:
why `httpId`? I thought it was a client or user ID

Reply:
ShowResult only for testing without using UI, and to one conclole can be sended two or more request, so I use httpId(probablu I need rename it to queryId) to see differend query result